### DISCUSSION TITLE
Install Akavardo

### DISCUSSION THREAD

**Discussion by MidhunMDas** [akvorado/akvorado#1505] [Q&A]
Tried to add conf in akvorado.yaml but getting error. Switch side its fine, tcpdump showing traffic,
Could you please help me to add my switch sflow agent IP

**Comment by MidhunMDas** [comment]
> Tried to add conf in akvorado.yaml but getting error. Switch side its fine, tcpdump showing traffic,
> Could you please help me to add my switch sflow agent IP 

I have added akvorado and inlet yaml files and configure snmp.
All containers are healthy and snmp check is successfull but flow is not showing.

Could you please help @vincentbernat

**Reply by zxyz** [reply]
Dear @MidhunMDas,

maybe you missed it, but the documentation has an elaborate tshoot part:

https://demo.akvorado.net/docs/troubleshooting#troubleshooting

This should help you getting started.

**Reply by MidhunMDas** [reply]
@zxyz heya, thanks for sharing. I will go through it and let you know
